### PR TITLE
develop to master (2024.07.08)

### PR DESCRIPTION
**Pull request by tkamizuka**
Major modifications are:
- adding gaussian ACE function to galcenimg.py
- bug fix on the FITS output function in galcenimg.py (modified to save image AFTER dark subtraction, noise addition, and digitization)
- Adding InP related data.
- modifying functions in runpixsim.py (added addnoise/digitize options)
- modifying save.py (added comments; modified to save psf data)
- bug fix on pixlight_analytic.py, pixlight_custom.py, and pixmulti.py (corrected pixel positions)
- modifying simpix_text.py

### CODE REVIEW

File: src/jis/binutils/runpixsim.py [hunk 1]
```diff
@@ -2,7 +2,7 @@
 import json
 import matplotlib.pylab as plt
 from jis.pixsim import simpix_stable as sp
-
+from jis.pixsim.addnoise import addnoise as AddNoise
```
Comment (astronasutarou) [inline src/jis/binutils/runpixsim.py:5]: 若干混乱しそうなので `pixsim_addnoise` or `addnoise_pixsim` とかにしておくのはどうでしょうか

File: src/jis/binutils/runpixsim.py [hunk 2]
```diff
@@ -61,19 +61,21 @@ def uniform_flat(detector):
...
-def init_images(control_params, detector, prior_dark = True):
+def init_images(control_params, detector, prior_dark = True, addnoise=True, digitize=True):
     """initialize pixcube.
 
     Args:
         control_params: control parameters
         detector: detector object
         prior_dark: if the dark is added (True) or not (False). default: True
+        addnoise: switch for noise-addition function (used when prior_dark=True).
+        digitize: swithc for digitize function (used when prior_dark=True).
```
Comment (astronasutarou) [inline src/jis/binutils/runpixsim.py:72]: s/swithc/switch/

**Review by astronasutarou — COMMENTED** [review]
Thank you for the PR. I left two minor comments.

**Review by HajimeKawahara — APPROVED** [review]
Thanks! I have no comment.

**Review by YuiKasagi — APPROVED** [review]
Thank you for the PR. I have no comments.

File: src/jis/binutils/runpixsim.py (hunk 2 above)
Comment (tkamizuka) [reply src/jis/binutils/runpixsim.py]: Thanks. Typo fixed (2024.7.21)

File: src/jis/binutils/runpixsim.py (hunk 1 above)
Comment (tkamizuka) [reply src/jis/binutils/runpixsim.py]: Thanks. I changed the name to pixsim_addnoise.

**Review by astronasutarou — APPROVED** [review]
Thank you! I approved the PR!

**Pull request merged**